### PR TITLE
fix: get the session timestamp from Spanner

### LINKED ISSUE #223: SpannerDb::timestamp should return CURRENT_TIMESTAMP()
SpannerDb inherited MysqlDb's behavior where it creates its self.timestamp() value from the current server's time.

For the sake of sanity we should be using `CURRENT_TIMESTAMP()` from Spanner itself instead (as the python does)

### PR DESCRIPTION
**Pull request by pjenvey**
and fix reservation of the std collection id space w/:

- a small adjustment to spanner's autoincrement
- a dummy value on mysql (nothing else is preserved between mysqld restarts)

and enable docker hub deploying

Closes #223, #231

### CODE REVIEW

File: db-tests/src/support.rs [hunk 1]
```diff
@@ -27,7 +27,11 @@ pub async fn db() -> Result<Box<dyn Db>> {
     };
 
     let pool = syncstorage::db::pool_from_settings(&settings)?;
-    pool.get().compat().await
+    let db = pool.get().compat().await?;
+    // Spanner won't have a timestamp until lock_for_xxx are called: fill one
+    // it for it
```
Comment (jrconlin) [inline db-tests/src/support.rs]: it's it

File: migrations/2018-08-28-010336_init/up.sql [hunk 2]
```diff
@@ -26,7 +26,7 @@ CREATE TABLE `bso` (
 CREATE TABLE `collections` (
     `id` INT PRIMARY KEY      NOT NULL AUTO_INCREMENT,
     `name` VARCHAR(32) UNIQUE NOT NULL
-) ENGINE=InnoDB AUTO_INCREMENT=100 DEFAULT CHARSET=latin1;
```
Comment (jrconlin) [inline migrations/2018-08-28-010336_init/up.sql]: Isn't it bad form to update an old migration?

File: migrations/2019-09-25-174347_min_collection_id/up.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+-- Reserve space for additions to the standard collections
+INSERT INTO collections (id, name)
```
Comment (jrconlin) [inline migrations/2019-09-25-174347_min_collection_id/up.sql]: stupid thought? We already filter for "xxx*" maybe use that for this?

File: migrations/2018-08-28-010336_init/up.sql (hunk 2 above)
Comment (pjenvey) [reply migrations/2018-08-28-010336_init/up.sql]: Definitely but with AUTO_INCREMENT being ephemeral it doesn't really matter -- which is an argument for leaving it -- but I'm nuking it so no one might mistake it for being needed/useful

**Pull request merged**